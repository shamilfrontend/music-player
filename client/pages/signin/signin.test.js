import { shallowMount } from '@vue/test-utils'

import Component from './index'

describe('SignInPage', () => {
  let instance

  beforeEach(() => {
    instance = shallowMount(Component, {
      stubs: ['nuxt-link']
    })
  })

  test('should match snapshot', () => {
    expect(instance.html()).toMatchSnapshot()
  })
})
