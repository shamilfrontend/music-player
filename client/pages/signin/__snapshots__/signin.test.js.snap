// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SignInPage should match snapshot 1`] = `
<div class="login-page">
  <div class="login-page__inner">
    <h1 class="login-page__title">Вход</h1> <input type="text" placeholder="Введите логин" class="login-page__field"> <input type="password" placeholder="Введите пароль" class="login-page__field"> <button class="login-page__btn">Войти</button>
    <nuxt-link-stub to="/signup" class="login-page__link">Регистрация</nuxt-link-stub>
  </div>
</div>
`;
